Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1637,6 +1638,49 @@ def get_mapping_fields_command(search_base):
 '''
 
 
+def set_password_not_expire(default_base_dn):
+    args = demisto.args()
+    sam_account_name = args.get('username')
+    pwd_n_exp = args.get('value')
+
```

Comment:
I would add here another variable for the sAMAccountName and use it in the next block.
For example: 
sam_account_name2 = args.get('sAMAccountName')